types/rtlcss/rtlcss-tests.ts

```diff
@@ -1,7 +1,9 @@
-import * as rtlcss from "rtlcss";
-import { Transformer } from 'postcss';
+import rtlcss = require('rtlcss');
```

Comment:
Any reason why we're changing the import syntax here?

Reply:
Oh, no particular one, just intent to align with now recommended syntax:
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require
it's for DT, not for consumers of the package,

Reply:
Oh no worries, good pickup :)